include/net/net_pkt.h

```diff
@@ -225,7 +225,7 @@ struct net_pkt {
 	uint16_t ipv6_prev_hdr_start;
 
 #if defined(CONFIG_NET_IPV6_FRAGMENT)
-	uint16_t ipv6_fragment_offset;	/* Fragment offset of this packet */
+	uint16_t ipv6_fragment_flags;	/* Fragment offset and M flag of this packet */
```

Comment:
Also here I would suggest `Fragment offset and M (More Fragment) flag of this packet`

Reply:
done